### PR TITLE
Use Marathon event stream directly for SI tests.

### PR DESCRIPTION
**Pull request by jeschkies**
Summary:
Some system integration tests SSH into the cluster and call the Marathon
event API from there. This is not necessary. We can call it from the
outside and assert each event.

Depends on #5860.

JIRA issues: MARATHON-7964

### CODE REVIEW

**Comment by jeschkies** [comment]
The info endpoint is failing.

**Comment by ichernetsky** [comment]
@jeschkies it is `Caused by: $file.ci.utils$BuildException: 'flake8 --count --max-line-length=120 tests/system src/test/python' exited with 1`, AFAICT.

**Comment by jeschkies** [comment]
> @jeschkies it is Caused by: $file.ci.utils$BuildException: 'flake8 --count --max-line-length=120 tests/system src/test/python' exited with 1, AFAICT.

Yep. The Python and thus flake8 version is out of date.

File: tests/Makefile [hunk 1]
```diff
@@ -17,7 +17,7 @@ init:
 	pipenv install --dev --skip-lock
 
 build:
-	flake8 --count --max-line-length=120
+	flake8 --count --max-line-length=120 system
```
Comment (alenkacz) [inline tests/Makefile]: hood catch! 👍

File: Jenkinsfile.pr.si [hunk 2]
```diff
@@ -21,6 +21,7 @@ node('py35') {
                 sh """git fetch origin pull/$params.Pull_Request_Id/head:$params.Pull_Request_Id"""
                 sh """git checkout $params.Pull_Request_Id"""
             }
+            sh """pip3 install requests"""
```
Comment (wavesoft) [inline Jenkinsfile.pr.si]: Is `requests` needed anywhere?

File: ami/install.bash [hunk 3]
```diff
@@ -1,46 +1,52 @@
...
-        npm \
-        python3-pip \
-        rpm \
-        sbt
+        openjdk-8-jdk \
+        libssl-dev \
+        sbt \
+        zlib1g-dev
+
+# Download, compile and install Python 3.6.2
```
Comment (wavesoft) [inline ami/install.bash]: How come install python from sources and not using a binary release? (not that it's changing that much, just curious).

File: Jenkinsfile.pr.si (hunk 2 above)
Comment (jeschkies) [reply Jenkinsfile.pr.si]: Yes. For Github statuses.

File: ami/install.bash (hunk 3 above)
Comment (jeschkies) [reply ami/install.bash]: There is no Python 3.6 backport for Stretch at least I did not find any.

**Pull request merged**